components/talksPage/talksPage.module.scss

```diff
@@ -0,0 +1,76 @@
+$itemGrow: 1.2;
+$duration: 250ms;
+
+.descriptionContainer {
+  display: flex;
+  gap: 20px;
+  flex-direction: column;
+  direction: ltr;
+  width: 80%;
+  margin: 0 auto;
+}
+
+.title {
+  font-weight: 800;
+  font-size: 36px;
+  color: #39475d;
+  line-height: 47px;
+}
+
+.wrapper {
+  display: grid;
+  grid-template-columns: repeat(4, 100%);
+  overflow: hidden;
+  height: 200px;
+  scroll-behavior: smooth;
+
+  .section {
+    width: 100%;
+    position: relative;
+    display: grid;
+    grid-template-columns: repeat(5, auto);
+    margin: 20px 0;
+
+    .item {
+      padding: 0 2px;
+      transition: $duration all;
+
+      &:hover {
+        margin: 0 40px;
+        transform: scale(1.2);
+      }
+    }
+
+    .arrow__btn {
```

Comment:
do we need this nested class or can it be outside the wrapper class thanks to css modules?